### PR TITLE
Fix executeCommand race condition

### PR DESCRIPTION
**Pull request by hristoterezov**
executeCommand for display name, avatarURL and email was not working if called right after the iframe is created.

### CODE REVIEW

File: react/features/base/participants/actions.js [hunk 1]
```diff
@@ -9,75 +9,6 @@ import {
 import { getLocalParticipant } from './functions';
 
 /**
- * Action to update a participant's avatar ID.
```
Comment (saghul) [inline react/features/base/participants/actions.js]: I'm -1 on this change. Each participant property might be changed from different places, so encapsulating it in an action makes sense to me.

IIRC @lyubomir added those, so I'll let him have the last say, but I'd like them to stay separate.

Comment (hristoterezov) [inline react/features/base/participants/actions.js]: I've discussed that with @lyubomir and he agreed on this approach.

The problem here is that we will need separate actions for the local participant in order to handle the use case when the local participant doesn't have an ID yet.

Comment (saghul) [inline react/features/base/participants/actions.js]: I think I get it.

**Comment by hristoterezov** [comment]
@saghul Since you've approved the PR could you please merge it?

**Pull request merged**